pysmt/test/test_optimization.py

```diff
@@ -15,13 +15,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 #
```

Comment:
Why adding imports if the file is unchanged? In general, try to see with VSCode if we can remove unused imports...